src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_SIZE = 10;
+    private Object[] keys = new Object[STORAGE_SIZE];
+    private Object[] values = new Object[STORAGE_SIZE];
+    private int size;
+    private int index;
+
+    public StorageImpl() {
+
+    }
+
+    public StorageImpl(Object[] keys, Object[] values, int size) {
+        this.keys = keys;
+        this.values = values;
+        this.size = size;
+    }
```

Comment:
According to Oleh's comment before:
```suggestion
    private static final int STORAGE_SIZE = 10;
    private K[] keys;
    private V[] values;
    private int size;
    
    public StorageImpl() {
        this.keys = (K[]) new Object[STORAGE_SIZE];
        this.values = (V[]) new Object[STORAGE_SIZE];
    }
```